v1/pipeline/steps/cd/k8s-traffic-routing-rule.yaml

```diff
@@ -1,4 +1,4 @@
-title: TrafficRouteRule
+title: K8sTrafficRoutingRouteRule
```

Comment:
The same as above.

Reply:
No. Title should be K8sTrafficRoutingRouteRule. We should actually change the file name as well.